src/components/BookItem.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const BookItem = () => (
+  <p>Under Construction</p>
```

Comment:
- [ ] Your book item component should be the one creating each book, receiving a book prop, and rendering the book title and author, please change your logic accordingly to match the project requirements.